packages/stark-ui/src/modules/app-sidebar/components/app-sidebar.component.spec.ts

```diff
@@ -1,26 +1,65 @@
+/* tslint:disable:completed-docs */
+import { Subject } from "rxjs";
 import { CommonModule } from "@angular/common";
 import { NoopAnimationsModule } from "@angular/platform-browser/animations";
 import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { BreakpointState, BreakpointObserver } from "@angular/cdk/layout";
 import { MatSidenavModule } from "@angular/material/sidenav";
-import { STARK_LOGGING_SERVICE } from "@nationalbankbelgium/stark-core";
-import { MockStarkLoggingService } from "@nationalbankbelgium/stark-core/testing";
+import { TransitionHookFn, TransitionStateHookFn, HookMatchCriteria } from "@uirouter/core";
+import {
+	STARK_LOGGING_SERVICE,
+	StarkLoggingService,
+	STARK_ROUTING_SERVICE,
+	StarkRoutingService,
+	StarkRoutingTransitionHook
+} from "@nationalbankbelgium/stark-core";
+import { MockStarkLoggingService, MockStarkRoutingService } from "@nationalbankbelgium/stark-core/testing";
 import { StarkAppSidebarComponent } from "./app-sidebar.component";
+import { STARK_APP_SIDEBAR_SERVICE, StarkAppSidebarService } from "../services/app-sidebar.service.intf";
 import { MockAppSidebarService } from "../testing/app-sidebar.mock";
-import { STARK_APP_SIDEBAR_SERVICE } from "../services/app-sidebar.service.intf";
-import { BreakpointState } from "@angular/cdk/layout";
 
-describe("AppSidebarComponent", () => {
-	let fixture: ComponentFixture<StarkAppSidebarComponent>;
-	let component: StarkAppSidebarComponent;
+// Definitions
+/**
+ * Defines the breakpoint set in `app-sidebar.component.ts`.
+ */
+const BREAKPOINT_STRING: string = "(min-width: 1280px)";
+const _fakeBreakPointObservable: Subject<BreakpointState> = new Subject<BreakpointState>();
+
+/**
+ * Use this function to mock the screen resizing (breakpoint triggering)
+ * {@link BreakpointObserver}
+ */
+function simulateBreakPointStateChange(breakPointState: BreakpointState): void {
+	_fakeBreakPointObservable.next(breakPointState);
+}
+
+let fixture: ComponentFixture<StarkAppSidebarComponent>;
+let component: StarkAppSidebarComponent;
+
+// Mocked services initialisation
```

Comment:
Same remark: shouldn't all the mock services be initiliazed in the `beforeEach()`?